analyzers/tests/SonarAnalyzer.Test/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Operations.cs

```diff
@@ -526,23 +526,31 @@ public void DynamicObjectCreation_SetsNotNull()
     [TestMethod]
     public void ObjectCreation_SetsNotNull()
     {
-        const string code = @"
-object assigned;
-var obj = new Object();
-var valueType = new Guid();
-var declared = new Exception();
-assigned = new EventArgs();
-
-Tag(""Declared"", declared);
-Tag(""Assigned"", assigned);
-Tag(""ValueType"", valueType);
-Tag(""Object"", obj);";
+        const string code = """
+            object assigned;
+            var obj = new Object();
+            var valueType = new Guid();
+            var declared = new Exception();
+            assigned = new EventArgs();
+            var collection1 = new List<int>();
+            collection1.Add(42);
```

Comment:
This fails to set the constraint from empty to non-empty, because it's part of the rule (for now)